File: week04-2.py
##week04-2.py 是今天的LeetCode挑戰是 Easy 題(二合一)
##LeetCode 2529. Maximum Count of Positive Integer and Negative Integer
class Solution:
    def maximumCount(self, nums: List[int]) -> int:
        pos = 0
        neg = 0 ##迴圈前面,準備好,都放0
        for i in range(len(nums)): ##迴圈裡,更新他
            if nums[i] > 0: pos += 1
            if nums[i] < 0: neg += 1
            ##迴圈後面,把答案拿來用
        return max(pos, neg)

File: week04-2.c
///week04-2.c 是今天的LeetCode挑戰是 Easy 題(二合一)
///LeetCode 2529. Maximum Count of Positive Integer and Negative Integer
int maximumCount(int* nums, int numsSize) {

    int pos = 0,neg = 0; ///迴圈前面都是0
    for(int i=0; i<numsSize; i++){
        if( nums[i] > 0 ) pos++; ///正數
        if( nums[i] < 0 ) neg++; ///負數
    }///迴圈中間,更新、修改他

    ///迴圈後面,把它拿來用
    if(pos>neg) return pos; ///正數比較多,
    else return neg; ///不然,就負數
}
